src/modules/menu/components/MenuSection.vue

```diff
@@ -28,7 +28,7 @@ export default {
         },
         showContent: {
             type: Boolean,
-            default: false,
+            default: true,
```

Comment:
With what you have done below, I'm not sure this is still relevant.
I'd prefer to have its default behavior to be hidden

Reply:
About that, maybe we could add something in its `:show-content=""` part, that would open it only if it is not the default topic (the first of the bunch).

This way the topic tree would open only if you selected another topic (which is how map.geo.admin.ch does it)